docs/dev/internal/payload-format.md

```diff
@@ -0,0 +1,30 @@
+# Payload format
+
+## Timespan
+
+### Example
+
+```json
+{
+    "time_unit": "milliseconds",
+    "value": 10
+}
+```
+
+## Timing Distribution
+
+### Example:
+
+```json
+{
+    "bucket_count": 100,
+    "range": [0, 60000],
+    "histogram_type": "expontential",
+    "sum": 3,
+    "time_unit": "milliseconds",
+    "values": {
+        "0": 1,
+        "1": 3,
+    }
+}
+```
```

Comment:
This file was maybe included by accident?

Reply:
It was...